contracts/RootChain.sol

```diff
@@ -271,15 +271,17 @@ contract RootChain {
      * @dev Processes any exits that have completed the challenge period.
      * @param _token Token type to process.
      */
-    function finalizeExits(address _token)
+    function finalizeExits(address _token, uint256 _topUtxoPos, uint256 _exitsToProcess)
         public
     {
         uint256 utxoPos;
         uint256 exitable_at;
+        uint256 _exitsLeft = _exitsToProcess;
         (utxoPos, exitable_at) = getNextExit(_token);
+        require(_topUtxoPos == utxoPos || _topUtxoPos == 0);
```

Comment:
May be useful to add some clarifying comments here if not planned for later.

Reply:
Added explanation in function docs.